services/myget/myget.tester.js

```diff
@@ -163,11 +164,11 @@ t.create('version (unexpected second response)')
   .intercept(nock =>
     nock('https://api-v2v3search-0.nuget.org')
       .get(
-        '/query?q=packageid:mongodb.driver.core&prerelease=true&semVerLevel=2'
+        '/query?q=packageid%3Amongodb.driver.core&prerelease=true&semVerLevel=2'
       )
       .reply(invalidJSON)
   )
-  .expectJSON({ name: 'mongodb', value: 'invalid' })
+  .expectJSON({ name: 'myget', value: 'unparseable json response' })
```

Comment:
The error messages now get the label from `defaultBadgeData`, which is why the MyGet error badges no longer show the feed name. I think this is fine.